src/dview/dvfilereader.cpp

```diff
@@ -1070,8 +1070,6 @@ bool wxDVFileReader::ReadSQLFile(wxDVPlotCtrl* plotWin, const wxString& filename
 			sqlite3_finalize(sqlStmtPtr);
 		}
 
-		bool isLeapYear = false;

```

Comment:
It does seem strange that this framework is in place for leap years, but that the local variable "year" is never used.  Perhaps an artifact of someone's incomplete development, we can remove for now.